ssg/build_yaml.py

```diff
@@ -131,6 +131,16 @@ def to_xml_element(self):
 
         return element
 
+    def get_rule_selectors(self):
+        return list(filter(lambda x: "=" not in x, self.selections))
+
+    def get_variable_selectors(self):
+        variables = dict()
+        for var_selection in filter(lambda x: "=" in x, self.selections):
```

Comment:
If there will be 2 different lists in profile YAML files (one for rules and second for values), then we won't need to create weird methods.

Reply:
I understand, but separating the variable selectors from rule selectors makes experience of crafting profiles a bit worse.
It is very convenient to have the variable selector close to the rule selector in profile yaml.